lib/src/main/java/graphql/nadel/schema/NadelDirectives.kt

```diff
@@ -223,7 +223,10 @@ object NadelDirectives {
                 val identifiedByValues = resolveArgumentValue<List<Any>>(inputIdentifiedBy)
                 val identifiedBy = createObjectIdentifiers(identifiedByValues)
 
-                val conditionalHydration = buildConditionalHydrationObject(directive.getArgument("when"))?.result
+                val conditionalHydration = directive.getArgument("when")
+                    ?.let {
```

Comment:
Handle case where schema does not define `when` and the value is null.

Reply:
makes sense, thanks for picking it up 👍